plugins/editor/class.editor.plugin.php

```diff
@@ -1014,7 +1014,7 @@ public function saveUploads($id, $type) {
             $categoryID = $discussion->CategoryID;
             $category = CategoryModel::categories($categoryID);
 
-            if ($category && $category['AllowFileUploads'] !==1  && Gdn::request()->getValue('MediaIDs') !== false) {
+            if ($category && $category['AllowFileUploads'] !=1  && Gdn::request()->getValue('MediaIDs') !== false) {
```

Comment:
Extra space to align with code standards
```suggestion
            if ($category && $category['AllowFileUploads'] != 1  && Gdn::request()->getValue('MediaIDs') !== false) {
```